output.json

```diff
@@ -0,0 +1 @@
+{"users":["user1","user2","user3"]}
```

Comment:
This output is not correct based on the input file, why you define a dummy array for users?